extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -96,14 +96,22 @@ export function maybeExecuteRealTimeConfig_(a4aElement, customMacros) {
   // For each vendor the publisher has specified, inflate the vendor
   // url if it exists, and send the RTC request.
   Object.keys(rtcConfig['vendors'] || []).forEach(vendor => {
-    const url = RTC_VENDORS[vendor.toLowerCase()];
+    const vendorObject = RTC_VENDORS[vendor.toLowerCase()];
+    const url = vendorObject ? vendorObject.url : '';
     if (!url) {
       return logAndAddErrorResponse_(promiseArray,
           RTC_ERROR_ENUM.UNKNOWN_VENDOR, vendor);
     }
+    const validVendorMacros = {};
+    Object.keys(rtcConfig['vendors'][vendor]).forEach(macro => {
+      if (vendorObject.macros && vendorObject.macros.includes(macro)) {
+        validVendorMacros[macro] = rtcConfig['vendors'][vendor][macro];
+      } else {
+        user().warn(TAG, `Invalid macro: ${macro} for vendor: ${vendor}`);
```

Comment:
Can we make this "Unknown macro..."?  I think that's much clearer otherwise pub may assume value was invalid

Reply:
done